.github/pull_request_template.md

```diff
@@ -2,7 +2,8 @@
 
 ### Checklist
 
-- [ ] Confirm that `run-checks` script has been executed.
+- [ ] Confirm that `run-checks` script has been executed. The first run
+may take around 15 minutes.
```

Comment:
I think this warning should be from the script its self. Besides, by the time, someone reaches the PR, they might have already ran it (we mention it also in CONTRIBUTING.md file, probably worth mentioning it there as well).

Reply:
Ok, I've just added a comment at the top of the shell scripts instead. Considered adding it in a print statement at the top of run-checks.rs, but fear that it will get missed as the text scrolls down. WDYT?